arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/raise/RaiseAccumulate.kt

```diff
@@ -521,10 +521,19 @@ public inline fun <Error, A> Raise<Error>.forEachAccumulating(
   iterator: Iterator<A>,
   combine: (Error, Error) -> Error,
   @BuilderInference block: RaiseAccumulate<Error>.(A) -> Unit
+): Unit = forEachAccumulatingImpl(iterator, combine) { item, _ -> block(item) }
+
+@PublishedApi @JvmSynthetic
```

Comment:
Synthetic is a nice touch here! Note that this technically doesn't 100% absolve us of ABI compatibility issues according to the discussion on https://github.com/Kotlin/binary-compatibility-validator/issues/165. I wish we had access to `@InlineOnly`, then maybe BCV would consider those methods to not be part of our ABI, but alas.